lib/chef/server_api.rb

```diff
@@ -31,6 +31,7 @@ def initialize(url = Chef::Config[:chef_server_url], options = {})
       options[:client_name] ||= Chef::Config[:node_name]
       options[:signing_key_filename] ||= Chef::Config[:client_key]
       options[:signing_key_filename] = nil if chef_zero_uri?(url)
+      options[:inflate_json_class] = false
```

Comment:
is this also going to wind up slurping in deprecation of json_class everywhere into the scope?